lib/util/esWrapper.js

```diff
@@ -114,63 +121,106 @@ class ESWrapper {
     }
 
     if (error instanceof es.errors.NoConnections) {
-      return new ServiceUnavailableError('Elasticsearch service is not connected');
+      errorsManager.throw(
+        'external',
+        'elasticsearch',
+        'elasticsearch_service_not_connected');
     }
 
     let message = error.message || '';
 
-    const messageReplaced = errorMessagesMapping.some(mapping => {
-      message = message.replace(mapping.regex, mapping.replacement);
-      return message !== error.message;
-    });
+    let kuzzleError = null;
+
+    for (const mapping of errorMessagesMapping) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Reduce the total number of "break" and "continue" statements in this loop to use one at most. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ATooManyBreakOrContinueInLoop)